bundles/org.openhab.binding.buienradar/src/main/java/org/openhab/binding/buienradar/internal/buienradarapi/BuienradarPredictionAPI.java

```diff
@@ -120,37 +123,53 @@ public final BigDecimal getIntensity() {
             }
 
             @Override
-            public final ZonedDateTime getDateTime() {
+            public ZonedDateTime getDateTimeOfPrediction() {
                 return dateTime;
             }
+
+            @Override
+            public ZonedDateTime getActualDateTime() {
+                // TODO Auto-generated method stub
```

Comment:
But I dont like this TODO / comment :-)

Reply:
grmbl... This really brings back memories to all the crappy auto-generated comments everywhere. I removed it.